deploy/configs/es-workaround.bash

```diff
@@ -70,7 +10,7 @@ spec:
     apiVersion: v1
     metadata:
       name: elasticsearch
-      namespace: "sa-telemetry"
+      namespace: ${SAF_PROJECT}
```

Comment:
It's my understanding that the certs below only work in `sa-telemetry` namespace. Should add 
`SAF_PROJECT=sa-telemetry` to the top of this file